examples/js-cloud-server/cloudflare-worker/tsconfig.app.json

```diff
@@ -0,0 +1,25 @@
+{
+    "extends": "./tsconfig.json",
+    "compilerOptions": {
+        "outDir": "../../../dist/out-tsc",
+
+        "target": "es2021" /* Set the JavaScript language version for emitted JavaScript and include compatible library declarations. */,
+        "lib": ["es2021"] /* Specify a set of bundled library declaration files that describe the target runtime environment. */,
+        "jsx": "react" /* Specify what JSX code is generated. */,
+        "module": "es2022" /* Specify what module code is generated. */,
+        "moduleResolution": "node" /* Specify how TypeScript looks up a file from a given module specifier. */,
+        "types": ["@cloudflare/workers-types"] /* Specify type package names to be included without being referenced in a source file. */,
+        "resolveJsonModule": true /* Enable importing .json files */,
+        "allowJs": true /* Allow JavaScript files to be a part of your program. Use the `checkJS` option to get errors from these files. */,
+        "checkJs": false /* Enable error reporting in type-checked JavaScript files. */,
+        "noEmit": true /* Disable emitting files from a compilation. */,
+        "isolatedModules": true /* Ensure that each file can be safely transpiled without relying on other imports. */,
+        "allowSyntheticDefaultImports": true /* Allow 'import x from y' when a module doesn't have a default export. */,
+        "forceConsistentCasingInFileNames": true /* Ensure that casing is correct in imports. */,
+
+        "strict": true /* Enable all strict type-checking options. */,
+        "skipLibCheck": true /* Skip type checking all .d.ts files. */
```

Comment:
Why all the comments?

Reply:
Why not comments? It's actually kinda nice having an explanation for each setting.
I'm assuming this is one of those flavors of JSON that allows comments, like the VS Code config files.

Reply:
These comments were all from the example app I generated using Cloudflare's CLI.

Reply:
We should do that for our cli too

Reply:
We should do that for our cli too, and maybe the sdk proxy too